cache/cache_helpers.py

```diff
@@ -21,6 +21,9 @@ def __init__(self, *, set_default: bool = False):
             self.defect_count = 0
             self.watcher_count = 0
 
+    def __str__(self) -> str:
+        return f'all_character_count: {self.all_character_count}, IC: {self.ironclad_count}, Silent: {self.silent_count}, Defect: {self.defect_count}, Watcher: {self.watcher_count}'
```

Comment:
Is this meant to be user-facing? If so, formatting could be improved a bit. If not, disregard this :)

Reply:
No I did this for debugging